_quarto.yml

```diff
@@ -29,92 +29,95 @@ website:
     contents:
       - href: index.qmd
         text: Welcome
-      - href: AGU2024.qmd
-        text: AGU 2024
       - text: "---"
-      - section: NASA VEDA Platform
+      - section: The VEDA Project 
```

Comment:
Not a new bug, but this section title isn't a link, but all the other one's are. Quirky inconsistency.

Reply:
Fixed - I made the "about" page the index of that section and the TOC heading now links to that.